driver/src/contracts/base_contract.rs

```diff
@@ -1,10 +1,18 @@
+extern crate ethereum_tx_sign;
```

Comment:
If I am not mistaken `extern crate` statements should no longer be needed in edition 2018 which seems to be used by the driver.

Reply:
Cool, let me remove all of those in a cleanup PR